### PR TITLE
fix image names

### PR DESCRIPTION
**Pull request by spaette**
fix typos in bitmap image file names

### CODE REVIEW

**Comment by spaette** [comment]
ping

@csalas-yubico

**Comment by spaette** [comment]
@emlun 

Can you inform of the status of this PR?

File: content/Developer_Program/WebAuthn_Starter_Kit/Installation/Images/mback19-change-execution-role-lambda-function-v1.jpg
Comment (csalas-yubico) [inline content/Developer_Program/WebAuthn_Starter_Kit/Installation/Images/mback19-change-execution-role-lambda-function-v1.jpg:1]: Changing this file name caused issues with Figure 19 on page /Developer_Program/WebAuthn_Starter_Kit/Installation/Manual_Back-End_Deployment.html

**Review by csalas-yubico — CHANGES_REQUESTED** [review]
There are breaking changes due to file name changes - see my comments for the issues found

File: content/Developer_Program/WebAuthn_Starter_Kit/Back-end_System_Design.adoc [hunk 1]
```diff
@@ -84,7 +84,7 @@ Database creation scripts available via GitHub (Source).
 The WebAuthn Starter Kit is includes 3 Lambda functions used as triggers by the Amazon Cognito Custom Authentication flow. Note that each Lambda service has a unique role, allowing it to only communicate with the other Lambda services related to the WebAuthn Authentication flow, and prohibiting communication with others.
 
 
-image::Images/back1-congnito-user-pool-lambda-triggers-v1.png[]
+image::Images/back1-cognito-user-pool-lambda-triggers-v1.png[]
```
[the remark this answers is not in the document]
Comment (spaette) [reply content/Developer_Program/WebAuthn_Starter_Kit/Back-end_System_Design.adoc:87]: Is it now loading correctly?

**Comment by spaette** [comment]
I presume rebasing is unnecessary.

File: content/Developer_Program/WebAuthn_Starter_Kit/Back-end_System_Design.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (csalas-yubico) [reply content/Developer_Program/WebAuthn_Starter_Kit/Back-end_System_Design.adoc:87]: Confirmed it's loading properly now

**Review by csalas-yubico — APPROVED** [review]
Changes are good - regarding your previous comments, I don't believe a rebase is necessary

**Pull request merged**